Makefile

```diff
@@ -44,9 +44,7 @@ services:
 # these are useful for devs
 jobs:
 	docker pull registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest
-	docker run -it -v "${CURDIR}/ci-operator:/ci-operator" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --from-dir /ci-operator/config --to-dir /ci-operator/jobs
-	docker pull registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest
```

Comment:
lets not do this change in this PR -- lets keep eventing only-bits in this PR for now

Reply:
Done.